changelog.d/issue-8951

```diff
@@ -0,0 +1,3 @@
+synopsis: Don't add `extra-prog-path: ~/.local/bin` when initially creating `~/.config/cabal/config`
```

Comment:
Non-blocking: I'd love a short description that would explain the benefit (which we all know, but that would save some pointer chasing for the casual reader in the changelog interface of the package index.)

Reply:
@sol: how about this extra explanation?